bridge/evm/test/BridgeLimiterTest.t.sol

```diff
@@ -148,12 +151,75 @@ contract BridgeLimiterTest is BridgeBaseTest {
+        assertEq(limiter.chainLimits(supportedChainID), 1000000000);
+    }
+
+    function testMultipleChainLimits() public {
+        // deploy new common contract with 2 supported chains
+        address[] memory _supportedTokens = new address[](4);
+        _supportedTokens[0] = wBTC;
+        _supportedTokens[1] = wETH;
+        _supportedTokens[2] = USDC;
+        _supportedTokens[3] = USDT;
+        uint8[] memory supportedChains = new uint8[](2);
+        supportedChains[0] = 11;
+        supportedChains[1] = 12;
+        common = new BridgeCommon(chainID, _supportedTokens, supportedChains);
+        // deploy new committee with new common contract
+        address[] memory _committee = new address[](5);
+        uint16[] memory _stake = new uint16[](5);
+        _committee[0] = committeeMemberA;
+        _committee[1] = committeeMemberB;
+        _committee[2] = committeeMemberC;
+        _committee[3] = committeeMemberD;
+        _committee[4] = committeeMemberE;
+        _stake[0] = 1000;
+        _stake[1] = 1000;
+        _stake[2] = 1000;
+        _stake[3] = 2002;
+        _stake[4] = 4998;
+        committee = new BridgeCommittee();
+        committee.initialize(address(common), _committee, _stake);
+        // deploy new limiter with 2 supported chains
+        uint64[] memory totalLimits = new uint64[](2);
+        totalLimits[0] = 10000000000;
+        totalLimits[1] = 20000000000;
+        uint256[] memory tokenPrices = new uint256[](4);
+        tokenPrices[0] = SUI_PRICE;
+        tokenPrices[1] = BTC_PRICE;
+        tokenPrices[2] = ETH_PRICE;
+        tokenPrices[3] = USDC_PRICE;
+        limiter = new BridgeLimiter();
+        limiter.initialize(address(committee), tokenPrices, supportedChains, totalLimits);
+        // check if the limits are set correctly
+        assertEq(limiter.chainLimits(11), 10000000000);
+        assertEq(limiter.chainLimits(12), 20000000000);
+        // check if the oldestChainTimestamp is set correctly
+        assertEq(limiter.oldestChainTimestamp(11), uint32(block.timestamp / 1 hours));
+        assertEq(limiter.oldestChainTimestamp(12), uint32(block.timestamp / 1 hours));
+
+        // check that limits are checked correctly
+        uint8 tokenID = 3;
+        uint256 amount = 999999 * 1000000; // USDC has 6 decimals
+        assertFalse(limiter.willAmountExceedLimit(11, tokenID, amount));
+        limiter.recordBridgeTransfers(11, tokenID, amount);
+        assertTrue(limiter.willAmountExceedLimit(11, tokenID, 2000000));
+        assertFalse(limiter.willAmountExceedLimit(11, tokenID, 1000000));
+
+        // check that transfers are recorded correctly
+        amount = 1100000 * 1000000; // USDC has 6 decimals
+        limiter.recordBridgeTransfers(12, tokenID, amount);
+        assertEq(
+            limiter.chainHourlyTransferAmount(
+                limiter.getChainHourTimestampKey(12, uint32(block.timestamp / 1 hours))
+            ),
+            11000000000
```

Comment:
check that chain 11 is clean (not impacted by chain 12)

Reply:
👍